src/Components/Scenes/ViewRequests/RequestsInbox/RequestsInbox-Container.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux';
+import RequestsInbox from './RequestsInbox';
+import { approverRequestsFetch } from '../../../../State/ViewRequests/actions';
+
+const mapStateToProps = ({ viewRequests }) => {
+  return {
+    requests: viewRequests.inboundRequests
+  };
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    approverRequestsFetch: approverEmail =>
```

Comment:
Incredibly stupid style nitpick, but did this wrap to two lines via `prettier` or something? It don't got no semicolon

Reply:
or curly boys

Reply:
Oh yeah that was definitely a prettier thing. I must have had it all on one line (no semicolon needed!) and it bumped it to two.